src/styles/styles.js

```diff
@@ -1579,6 +1579,37 @@ const styles = {
         width: 80,
         height: 80,
     },
+    chatAvatarWraper: {
```

Comment:
Spelling: this should be `Wrapper`

Reply:
Also, just so I understand, is this what adds the white border around each avatar? Why not just give each view a border instead of doing this method?

Reply:
removed the wrapper and added border

Reply:
Great, just wanted to confirm that the border width should be 4px.